### PR TITLE
Remove snap to item. Add caching for scrolling speedup

### PR DESCRIPTION
**Pull request by vladest**
https://highfidelity.fogbugz.com/f/cases/5525/Attachment-Window-Scrolls-Very-Slowly

Test plan:
- open Tablet
- goto Avatar->Attachments
- add some more attachments
- note if the scrolling speed/smoothness is better then in previous builds

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2293/HighFidelity-Beta-PR11123-3a013bf86e6071ddaa1a70d428fc084fe5f54781.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2293/HighFidelity-Beta-PR11123-3a013bf86e6071ddaa1a70d428fc084fe5f54781.exe

File: interface/resources/qml/hifi/dialogs/content/AttachmentsContent.qml [hunk 1]
```diff
@@ -50,7 +50,7 @@ Item {
                         margins: 4
                     }
                     clip: true
-                    snapMode: ListView.SnapToItem
+                    cacheBuffer: 4000
```
Comment (sethalves) [inline interface/resources/qml/hifi/dialogs/content/AttachmentsContent.qml]: Do we know what the default value for this is, on Windows?

Comment (vladest) [inline interface/resources/qml/hifi/dialogs/content/AttachmentsContent.qml]: according to docs its 2x delegate size

**Comment by cain-kilgore** [comment]
Testing.

**Comment by cain-kilgore** [comment]
# Test Failed
Test | Result
-------- | --------
open Tablet | Pass
goto Avatar->Attachments | Pass
add some more attachments | Pass
note if the scrolling speed/smoothness is better then in previous builds | Pass

## Fail Notes
The slider doesn't appear to be scrolling the contents while still holding down on it.



Let me know once this needs further testing.

###### Intel i5 7600K @ 4Ghz, 16GB DDR4 2133Mhz Corsair Vengeance, Asus Strix GTX 1050Ti 3GB, HTC Vive

**Comment by vladest** [comment]
@cain-kilgore it works slightly different: when you drag the slider there is a delay before content started to slide. I dont know the reason for this, but it was like this before
second - need check sliding smoothness only, rest of behavior was not changes

**Comment by cain-kilgore** [comment]
@vladest Sliding Smoothness is great. The gif above doesn't show it off due to the framerate but it's very smooth for me w/out recording.

**Comment by vladest** [comment]
@cain-kilgore then, probably, goal for this PR is achieved

**Comment by cain-kilgore** [comment]
@vladest All good! :)

**Pull request merged**